src/com/projectkorra/projectkorra/airbending/flight/AirFlight.java

```diff
@@ -108,20 +109,24 @@ public void progress() {
 		if (!bPlayer.canBend(this)) {
 			remove();
 			return;
-		} else if (!player.isSneaking() && !isHovering && !firstProgressIteration) {
```

Comment:
The `else if` formatting here is incorrect as well. Please change it back to how it was.